### ISSUE TITLE
[react-ui-theme] issue with @fontsource-variable

### ISSUE THREAD

**Issue by apoeco** [dxos/dxos#7369]
Greetings everyone :wave:
Thank you for this very interesting project, keep it up!
While I was trying to follow [your guides](https://docs.dxos.org/guide/getting-started.html#project-templates) to scaffold a new app, I encountered the following issue:

Steps to reproduce:
1. Run
```sh 
npm create @dxos@latest
cd my-app
git init . && git add . && git commit -m init
# The above should be unrelated, but I initialized a git repo because I saw the above error/warning 
# fatal: not a git repository (or any of the parent directories): .git
npm i
npm serve
```
2. Visit `http://localhost:5173/`

Error stacktrace:
```sh
[vite] Internal server error: [postcss] ENOENT: no such file or directory, open '@fontsource-variable/inter/slnt.css'
  Plugin: vite:css
  File: <omitted>/node_modules/@dxos/react-ui-theme/dist/plugin/node/theme.css:undefined:NaN
```

I have tried installing `@fontsource-variable/inter` but I had no luck :(
The above is the output of 
```sh
la node_modules/@fontsource-variable/inter/
```
```sh
 CHANGELOG.md
 files
 index.css
 LICENSE
 metadata.json
 opsz.css
 opsz-italic.css
 package.json
 README.md
 scss
 standard.css
 standard-italic.css
 unicode.json
 wght.css
 wght-italic.css
```

```
node: v18.17.1
npm: 9.6.7
```

I have also tried the above using `v.0.6.3` but I encountered the same issue.
I have also deployed to `vercel` and the build failed with the same issue.
Hope this helps :)

**Comment by apoeco** [comment]
Also worth noting that if I try the above with `pnpm`,
```sh
pnpm create @dxos@latest
cd my-app
pnpm i
```
I get the above `swc`? related error:
```sh
node_modules/.pnpm/fs-ext@2.0.0/node_modules/fs-ext: Running install script...
 ELIFECYCLE  Command failed.
node_modules/.pnpm/@swc+core@1.7.5_@swc+helpers@0.5.12/node_modules/@swc/core: Running postinstall script...
```

**Comment by wittjosiah** [comment]
@apoeco thanks for reporting. I think this will potentially be resolved in the next release via #7370. In the mean time, if you're using pnpm you could override the version that it's installing and that may resolve the issue.

**Comment by thure** [comment]
Resolved by #7370
